stats.go

```diff
@@ -76,7 +77,13 @@ func (s *statsExporter) submitMetric(v *view.View, row *view.Row, metricName str
 
 	switch data := row.Data.(type) {
 	case *view.CountData:
-		return client.Gauge(metricName, float64(data.Value), opt.tagMetrics(row.Tags, tags), rate)
+		// get a unique string for metric and associated tags
+		metricID := metricRowID(row, metricName)
+		// compute the difference of now and last collected period
+		submitData := data.Value - s.countData[metricID]
```

Comment:
Do we have an issue with the first iteration? The first time we do a `CountData`, `s.countData[metricID]` was init before right?

Reply:
Rather, `s.countData` is initialized via `make` in L48. if a key does not exist in the map, it gets the value type's (`int64` in this case) *zero value*, which is `0`

https://blog.golang.org/maps#TOC_3.